template.yaml

```diff
@@ -12,9 +11,9 @@ Parameters:
     Type: AWS::SSM::Parameter::Value<String>
     Description: URI to the OAUTH endpoint of the Cognito client
     Default: /NVA/CognitoUri
-  FsConfig:
-    Type: String
-    Default: "{{resolve:secretsmanager:fs-config}}"
+  FsConfigSecretName:
```

Comment:
Why do we need to create a parameter for the secret name? Is it likely that we will change the name of the secret?

Reply:
I agree. It will likely never change. I am removing it.